### PR TITLE
Increases monkeycap to 64

### LINKED ISSUE #36656: It is super easy to shut down all monkey cubes
1. Get 20 monkey cubes.
2. Dump them somewhere nobody will ever come and activate them.
3. Nobody can now use monkey cubes for the rest of the round.

They don't even need to be alive.

### PR DESCRIPTION
**Pull request by praisenarsie**
:cl: Frozenguy5
tweak: The monkeycap has been increased to 64.
/:cl:

Closes #36745 due to this being an alternative.
Fixes #36656

### CODE REVIEW

**Comment by CPTANT** [comment]
64 is still way too low and still a balance change. The server can handle hundreds of monkeys, even more now their behaviour is optimised.

**Comment by Kromgar** [comment]
200 would be a decent cap. I think 300 is doable as well...


Because i did it once when i finished xeno and was fucking around super bored

**Comment by fludd12** [comment]
20 was insufficient only sometimes for xenobio, and this more than triples the amount. We can test and increase it more after the freeze, but this seems like it'll hold for the time being

**Comment by CPTANT** [comment]
@fludd12  You are still pretending like mild monkey spam isn't a feature. You SHOULD be able to drop a couple of dozen monkeys in a hallway for who knows why.

**Comment by Armhulen** [comment]
freeze

**Comment by fludd12** [comment]
I think this should pass the freeze, since the limit was mostly a result of the horrible lag monkeycode caused before the refactor

**Comment by praisenarsie** [comment]
@Armhulen 
I got permission from naksu.

Also, why do you really need 200 monkeys. Literally what purpose.

**Comment by fludd12** [comment]
Six slime pens, two slimes each, set up to feed two generations takes 24 monkeys, leaving only 40 for the rest of the station.

**Comment by KorPhaeron** [comment]
If 64 is still too low we can increase it again later

**Comment by Repukan** [comment]
Five more

**Comment by CPTANT** [comment]
>If 64 is still too low we can increase it again later

Or we can you know, just make it 200 because we know that doesn't crash the server anyway.

**Comment by arsserpentarium** [comment]
@KorPhaeron , maybe we shouldn't nerf it to ground completely at first?

**Comment by KorPhaeron** [comment]
Why do people keep talking about "nerfs" when the fucking thing was crashing the server

**Comment by CPTANT** [comment]
Because people keep proposing a cap that is so low that it has nothing to do with server stability any more. 

From a stability perspective a cap lower than 200-300 is nonsense and that was before optimisation.

**Comment by Iamgoofball** [comment]
@CPTANT you do not know what you are talking about

**Comment by arsserpentarium** [comment]
Yes, he know. I would tell, that somrhow bad lags starts at 350 monkeys. So it should be a cap.

File: code/controllers/subsystem/mobs.dm [hunk 1]
```diff
@@ -7,7 +7,7 @@ SUBSYSTEM_DEF(mobs)
 	var/list/currentrun = list()
 	var/static/list/clients_by_zlevel[][]
 	var/static/list/cubemonkeys = list()
-	var/cubemonkeycap = 20
+	var/cubemonkeycap = 64
```
Comment (ShizCalev) [inline code/controllers/subsystem/mobs.dm]: how about just making it a damn config option

**Comment by CPTANT** [comment]
256, final offer

File: code/controllers/subsystem/mobs.dm (hunk 1 above)
Comment (praisenarsie) [reply code/controllers/subsystem/mobs.dm]: @vuonojenmustaturska

Comment (vuonojenmustaturska) [reply code/controllers/subsystem/mobs.dm]: If the headmins want it, we can do it later

**Comment by CPTANT** [comment]
64 still makes literally no sense from a server stability perspective

**Comment by PKPenguin321** [comment]
What the hell where did we get 64 from that's super arbitrary
Why not 79 or 52

**Comment by Tlaltecuhtli** [comment]
1 stack of monkey blocks

**Comment by Jalleo** [comment]
@PKPenguin321 possibly because its a nice round number in binary?

**Comment by Iamgoofball** [comment]
Multiple of 2

**Pull request merged**